examples/v1alpha2/traffic-matching-tcp.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
I think this should actually be in `examples/v1alpha2/experimental` and we'd also need a corresponding update in https://github.com/kubernetes-sigs/gateway-api/blob/master/hack/verify-examples-kind.sh#L63 that only tried to install experimental examples when `CHANNEL == "experimental"`.

Reply:
things have been moved to `experimental` as you suggested